packages/driver/src/cy/commands/actions/attachFile.ts

```diff
@@ -0,0 +1,315 @@
+import { basename } from 'path'
+import _ from 'lodash'
+
+import $dom from '../../../dom'
+import $errUtils from '../../../cypress/error_utils'
+import $actionability from '../../actionability'
+import { addEventCoords, dispatch } from './trigger'
+
+/* dropzone.js relies on an experimental, nonstandard API, webkitGetAsEntry().
+ * https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem/webkitGetAsEntry
+ * The behavior here varies between browsers, and though unpleasant, we have to attempt
+ * to replicate it as far as possible.
+ *
+ * In webkit browsers, item.webkitGetAsEntry() returns null, because our File()
```

Comment:
Great rundown! I wonder what this means as `webkitGetAsEntry` potentially moves to `getAsEntry` for dropzone.js and how it impacts this functionality

Reply:
I hope that when it is implemented, it'll be a bit more standard across browsers. The major browsers have been pretty good about that these days for non-prefixed methods. Not perfect, but pretty good.